job/server/pom.xml

```diff
@@ -124,6 +123,22 @@
         </execution>
       </executions>
     </plugin>
+
+    <!-- REST API documentation -->
+    <plugin>
+      <groupId>com.github.kongchen</groupId>
+      <artifactId>swagger-maven-plugin</artifactId>
+      <configuration>
+        <apiSources>
+          <apiSource>
+            <locations>alluxio.master</locations>
+            <templatePath>${project.parent.parent.basedir}/templates/strapdown.html.hbs</templatePath>
```

Comment:
seems like a provided file rather than a generated file; should templates/ be ignored? is this the entrypoint to styling the output HTML to whatever we like?

`project.parent.parent`... o.O

Reply:
Good catch on the templates, and yeah its because this is nested in two modules.